config.go

```diff
@@ -210,6 +210,8 @@ type Config struct {
 	MaxPendingChannels int    `long:"maxpendingchannels" description:"The maximum number of incoming pending channels permitted per peer."`
 	BackupFilePath     string `long:"backupfilepath" description:"The target location of the channel backup file"`
 
+	FeeURL string `long:"feeurl" description:"Optional URL for external fee estimation. If an URL is not specified, static fees will be used for estimation."`
```

Comment:
description is wrong, the fee estimation will depend on the chosen backend.

Reply:
fixed